nb/regression/trinkets/honeycomb.md

```diff
@@ -0,0 +1,15 @@
+```JavaScript
+Hexagon(4)
```

Comment:
This looks really nice written

Reply:
Yeah, it's getting a bit better to read. :)